main.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python
```

Comment:
@chatcody Write a description for this file, formatted as code to include at the top of the file

Reply:
Please write that description